test/ex_bitmex/rest/orders/cancel_bulk_test.exs

```diff
@@ -106,4 +106,61 @@ defmodule ExBitmex.Rest.Orders.CancelBulkTest do
              ]
     end
   end
+
+  test ".cancel_bulk timeout" do
+    use_cassette "rest/orders/cancel_bulk_timeout" do
+      {:ok, order, _} =
+        ExBitmex.Rest.Orders.create(
+          @credentials,
+          %{
+            symbol: "XBTUSD",
+            side: "Buy",
+            orderQty: 1,
+            price: 1
+          }
+        )
+
+      assert order == %ExBitmex.Order{
+               account: 158_898,
+               order_qty: 1,
+               multi_leg_reporting_type: "SingleSecurity",
+               exec_inst: "",
+               ord_rej_reason: "",
+               triggered: "",
+               price: 1,
+               simple_order_qty: nil,
+               time_in_force: "GoodTillCancel",
+               currency: "USD",
+               ord_status: "New",
+               text: "Submitted via API.",
+               simple_leaves_qty: nil,
+               cl_ord_id: "",
+               contingency_type: "",
+               side: "Buy",
+               ord_type: "Limit",
+               avg_px: nil,
+               cl_ord_link_id: "",
```

Comment:
Why do you asserting it here? It doesn't relate to `timeout` test

Reply:
This is not relate to the timeout at all. I just assert the whole object. That's all

Reply:
But why?

Reply:
Ah I know what you mean, I assert to make sure that order created success. Then next step is I want to cancel that order.

Reply:
Why not just assert on timeout?

Reply:
Can, then I just can simply mock the delete request. Right now I mock both.